flow/src/org/labkey/flow/view/GraphColumn.java

```diff
@@ -166,16 +166,20 @@ public void renderGraph(RenderContext ctx, Writer out) throws IOException
 
         if (showGraphs(ctx) == FlowQuerySettings.ShowGraphs.Inline)
         {
+            PageConfig pageConfig = HttpView.currentPageConfig();
+            String id = pageConfig.makeId("img_");
+
             SPAN(
                 at(style, "display:inline-block; vertical-align:top; height:" + graphSize + "; width:" + graphSize),
                 objectId == null ?
                     SPAN(cl("labkey-disabled labkey-flow-graph"), "No graph for:", BR(), graphSpec) :
                     IMG(
                         at(alt, "Graph of: " + graphSpec).at(title, graphSpec).at(style, "height: " + graphSize + "; width: " + graphSize)
                             .at(src, urlGraph(objectId, graphSpec, ctx.getContainer()))
-                            .at(onerror, "flowImgError(this);").cl("labkey-flow-graph")
+                            .id(id).cl("labkey-flow-graph")
                     )
             ).appendTo(out);
+            pageConfig.addHandler(id, "error", "flowImgError(this);");

```

Comment:
especially this one, since it is a DisplayColumn and can be in a grid.

Reply:
I guess I can just do an ensure pattern in renderGraph()